tests/unit/test_charm.py

```diff
@@ -7,13 +7,14 @@
 
 
 class TestCharm(unittest.TestCase):
-    def setUp(self) -> None:
+    def setUp(self):
         self.harness = Harness(SlurmdCharm)
         self.addCleanup(self.harness.cleanup)
         self.harness.begin()
 
     @patch("ops.framework.EventBase.defer")
     def test_check_etcd_fail(self, defer):
+        """Test check_etcd method failure behavior."""
```

Comment:
This comment counts for all the docstrings - I am cool with the docstrings. I will also probably need to follow suite and update my grammar. We don't want to be pulling in separate directions given that it is only the two of us!

Reply:
If I'm understanding your comment correctly, the convention (per D401) is for docstrings to use the imperative mood - so the highlighted example here is correct :)

Reply:
We were having a discussion with Omnivector regarding the mood to use for the docstrings. PEP 257 recommends using the imperative mood as a convention, not as law if you know what I mean. Me and David's docstrings defer slightly - this discussion happened after Omni accepted my initial pull requests - so I need to go back and change my docstrings to the imperative mood.

Reply:
Resolving this convo.